src/tracking-internal.js

```diff
@@ -82,8 +94,35 @@
 	 * @returns {void}
 	 */
 	function trackPageView(context) {
+		const sessionCookieSplit = `; ${document.cookie}`.split('; tracking_session_id='),
+			pvNumberCookieSplit = `; ${document.cookie}`.split('; pv_number='),
+			pvNumberGlobalCookieSplit = `; ${document.cookie}`.split('; pv_number_global='),
```

Comment:
what do you think about extracting it to a function that returns a proper string?